deisctl/deisctl.go

```diff
@@ -32,6 +32,7 @@ Commands, use "deisctl help <command>" to learn more:
   install           install components, or the entire platform
   uninstall         uninstall components
   list              list installed components
+  list-machines     Enumerate the current hosts in the cluster
```

Comment:
Since we're already a bit different between fleetctl and deisctl... would `machines` be a better choice than `list-machines`?